iterative-walkthrough/deployment-2/guestbook-redis/main.go

```diff
@@ -1,74 +1,111 @@
 package main
 
 import (
-    "log"
-    "net/http"
-    "os"
-    "github.com/zpatrick/fireball"
-    "github.com/garyburd/redigo/redis"
+	"fmt"
```

Comment:
Interesting - running `gofmt -w` doesn't seem to introduce a change to the file.